README.md

```diff
@@ -1,20 +1,18 @@
 # Ghidra Software Reverse Engineering Framework
 
-Ghidra is a software reverse engineering (SRE) framework created and maintained by the [National Security Agency][nsa] Research Directorate. This framework includes a suite of full-featured, high-end software analysis tools that enable users to analyze compiled code on a variety of platforms including Windows, Mac OS, and Linux. Capabilities include disassembly, assembly, decompilation, graphing, and scripting, along with hundreds of other features. Ghidra supports a wide variety of process instruction sets and executable formats and can be run in both user-interactive and automated modes. Users may also develop their own Ghidra plug-in components and/or scripts using Java or Python.
+Ghidra is a software reverse engineering (SRE) framework created and maintained by the [National Security Agency][nsa] Research Directorate. This framework includes a suite of full-featured, high-end software analysis tools that enable users to analyze compiled code on a variety of platforms including Windows, Mac OS, and Linux. Capabilities include disassembly, assembly, decompilation, graphing, and scripting, along with hundreds of other features. Ghidra supports a wide variety of process instruction sets and executable formats and can be run in both user-interactive and automated modes. Users can also develop their own Ghidra plug-in components and/or scripts using Java or Python.
 
-In support of NSA's Cybersecurity mission, Ghidra was built to solve scaling and teaming problems on complex SRE efforts, and to provide a customizable and extensible SRE research platform. NSA has applied Ghidra SRE capabilities to a variety of problems that involve analyzing malicious code and generating deep insights for SRE analysts who seek a better understanding of potential vulnerabilities in networks and systems.
+In support of the NSA's Cybersecurity mission, Ghidra was built to allow scaling and collaboration on complex SRE efforts, and to provide a customizable and extensible SRE research platform. The NSA has applied Ghidra SRE to a variety of problems that involve analyzing malicious code and generating deep insights for SRE analysts who seek a better understanding of potential vulnerabilities in networks and systems.
```

Comment:
You will see at [nsa.gov](https://www.nsa.gov/) there is no mention of “the” NSA. It is an all-too-common grammar error, frequently repeated by major news publications; and it like incorrectly calling CIA “the” CIA.

It’s perhpas better to suggest “solve scaling problems and improve collaboration.” And I think capabilities should remain untouched.

Reply:
The pedantry of defending obvious grammatical errors. Of course it's all the major news publications that are wrong!

> Hint: nsa.gov, ctrl+f, "the nsa"

Reply:
Well, I wasn't born in ~~the~~ USA so I strive to learn the code(s) but American English has unusually permissive operator overloading (e.g. "different than"), plus the heads of three-letter-abbreviation intelligence organizations never correct senators who ask, "did the XYZ investigate...". [Here's](https://www.washingtonpost.com/opinions/james-comey-republicans-are-wrong-transparency-is-possible-in-the-mueller-investigation/2019/03/04/f7e95f38-3ebc-11e9-a0d3-1210e58a94cf_story.html) Jim Comey acknowledging his non-insider status at "the FBI" earlier this week. 

Now, if anyone starts referring to "NSA" as an acronym, we'll have a discussion about sets and subsets.

Reply:
While it’s acceptable to refer to the NSA with “the,” It is also not always correct to add it.

For example, I wouldn’t refer to Brazil as “the Brazil.” 

As in, “I visited the Brazil last year; the Brazil was great.” — this is clearly incorrect grammar.

And yet, “I visited the NSA last year; the NSA was great.” — this is **incorrectly** considered good grammar.

Instead, to perhaps belabor the point, “I visited NSA last year; NSA was great.”

That is correct in the same way that, “I visited Brazil last year; Brazil was great” is actually the correct grammar.

Reply:
If you curl+f “the NSA” on nsa.gov, you’ll find at least one occurance of “the NSA” there. However, if you look up “NSA,” you’ll find many more other references with the article. 

I would invite those who are curious to look at this and think about why that is.

Reply:
This will be my last comment regarding this point, I promise.

If I said “the Bob wrote some code,” it’s arguably correct — though I don’t think so — but awkward.

Instead the phrase “Bob wrote some code,” is not awkward, and, therefore, much more correct.

“Bob” is a proper name of an entity. “NSA” is also the proper name of an entity.